### DISCUSSION TITLE
КД АБИЛОК КСЕН

### DISCUSSION THREAD

**Discussion by lorianss** [Cosmic-Overlord/TerraGov-Marine-Corps#318] [Ideas]
БЛЯ ВОТ БЫ НАХУЙ
ИКОНКА У КСЕН
МЕНЯЛАСЬ
ЕСЛИ КД ПОЧТИ ПРОШЛО
ВОТ БЫ БЫЛО БЫ АХУЕННО

**Comment by fe-ed** [comment]
ЧЕ, НИЧЕ НЕ ПОНЯЛ
